packages/icon-button/index.js

```diff
@@ -0,0 +1,107 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import withRipple from '@material/react-ripple';
+import {MDCIconButtonToggleFoundation} from '@material/icon-button';
+import IconToggle from './IconToggle';
+
+const {strings} = MDCIconButtonToggleFoundation;
+
+class IconButtonBase extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      classList: new Set(),
+      [strings.ARIA_PRESSED]: props[strings.ARIA_PRESSED],
```

Comment:
Do we need to do the same thing for `ARIA_LABEL`?

Reply:
aria-label was removed from mdc web's icon-button